### ISSUE TITLE
Browse for folder - option to specify directory

### ISSUE THREAD

**Issue by mirddes** [thiagodesacosta/HPIZ-Archiver#4]
![image](https://github.com/user-attachments/assets/4a9616ff-2f57-42f2-b64d-f0c82a4c1ef0)

i'd just really appreciate the ability to paste the location instead of manually navigating to it each time.

failing that defaulting to where hpiza is located would greatly reduce the apm required to navigate.

<3

**Comment by THGSCST** [comment]
I agree that the FolderBrowserDialog is outdated. This comes from the Net Framework, which Microsoft forgot to update for this specific dialog after Windows XP. I have some ideas on how to fix this in the next version.

For now, see the beta version: https://github.com/THGSCST/HPIZ-Archiver/releases/tag/r1.3.1b

**Comment by THGSCST** [comment]
Fixed in v1.4
